pkg/monitoring/pb/data-feeds/on-chain/registry/decode.go

```diff
@@ -89,28 +91,28 @@ func newFeedUpdated(
 		BlockTimestamp: m.BlockTimestamp,
 
 		// Execution Context - Source
-		MetaSourceId: m.MetaSourceId,
+		MetaSourceId: m.ExecutionContext.MetaSourceId,
```

Comment:
Wait, how come we didn't extract the `ExecutionContext` in the FeedUpdated proto, like we did for WriteConfirmed. Would that be a follow up?

Reply:
Please scope out a follow up for this - we should share the metadata proto definition across these events

Reply:
Followup ticket: https://smartcontract-it.atlassian.net/browse/PLEX-1492